packages/.eslintrc.js

```diff
@@ -23,7 +23,7 @@ module.exports = {
     'prettier/prettier': 'error',
     'filenames/match-regex': ['error', '^[a-z-.]+$', true],
 
-    complexity: ['error', { max: 19 }],
+    complexity: ['error', { max: 18 }],
```

Comment:
please try reduce it as much as possible in case the reducer was our most complex function

Reply:
Silly question... what does this `max` value mean?

Reply:
As far as I know, this is the maximum number of "paths of execution" allowed by the linter within a block statement, so every statement (if, else if, else, switch/case) will contribute with 1 to this complexity counter.

Reply:
in our current reducer we had 18 cases + 1 default, so this was the reason which made the linter to complain.